tensorflow/core/kernels/strided_slice_op.cc

```diff
@@ -529,7 +529,6 @@ REGISTER_KERNEL_BUILDER(Name("ResourceStridedSliceAssign")
 REGISTER_KERNEL_BUILDER(Name("TensorStridedSliceUpdate")
                             .Device(DEVICE_GPU)
                             .TypeConstraint<int32>("T")
-                            .HostMemory("input")
```

Comment:
I think the int32 gpu kernel should have hostmemory annotations to be compatible with the other int32 gpu kernels

Reply:
You're right. I reverted that change.